src/services/payment.service.ts

```diff
@@ -414,10 +415,12 @@ export class PaymentService {
     additionalInvoiceOptions?: Partial<Stripe.InvoiceCreateParams>;
   }): Promise<PaymentIntent> {
     let couponId: string | undefined = undefined;
+    const isCryptoCurrency = this.bit2MeService.isAllowedCurrency(currency);
+    const currencyForStripeInvoice = isCryptoCurrency ? 'eur' : currency;
```

Comment:
Does the Bit2Me format coincide with the Stripe's one? I would add a layer between that, otherwise, Bit2Me changing the API currency format will break the process

Reply:
Good catch! I've verified the formats:

- **Stripe**: Uses lowercase ISO 4217 codes (eur, usd)
- **Bit2Me**: Uses uppercase format (EUR, USD, BTC, ETH)

Working on it! 🚀